### PR TITLE
Updating Azure Media Player

### PR DESCRIPTION
**Pull request by sebastienros**
Initial commit. Needs to remove previous scripts if not necessary anymore but I need confirmation.

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.Azure.MediaServices/Orchard.Azure.MediaServices.csproj [hunk 1]
```diff
@@ -147,6 +147,8 @@
     <Reference Include="System.Xml.Linq" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Assets\Css\azuremediaplayer.min.css" />
+    <Content Include="Assets\JavaScript\Lib\azuremediaplayer.min.js" />
```
Comment (DanielStolt) [inline src/Orchard.Web/Modules/Orchard.Azure.MediaServices/Orchard.Azure.MediaServices.csproj]: We shouldn't commit minified resources in the repo. Instead commit dev version and let the gulp pipeline handle the minification, as previously agreed.

Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Azure.MediaServices/Orchard.Azure.MediaServices.csproj]: Correction: minified versions should be handled by gulp, I agree, though they will still be committed in the repos.
I will try to find the non-minified version for these files.

**Comment by sebastienros** [comment]
- [ ] Add non-minified version of scripts
- [ ] Remove injector pipeline

**Comment by sebastienros** [comment]
Please fix the merge issues.

**Comment by Jetski5822** [comment]
@sebastienros you are the one that needs to fix merge issues :) haha

**Comment by TFleury** [comment]
It seems the module is outdated. I opened an issue to discuss that #7671

**Comment by sebastienros** [comment]
Closing it in favor of https://github.com/OrchardCMS/Orchard/pull/7618

**Pull request closed without merging**